packages/bot/src/stores/toolbar-store.js

```diff
@@ -15,24 +16,18 @@ export default class ToolbarStore {
     @observable is_search_loading = false;
     @observable file_name = localize('Untitled Bot');
 
-    @action.bound
-    onRunClick() {
-        this.root_store.run_panel.onRunButtonClick();
-    }
-
-    @action.bound
-    onStopClick() {
-        this.root_store.run_panel.onStopButtonClick();
-    }
-
     @action.bound
     onToolboxToggle() {
-        const workspace      = Blockly.derivWorkspace;
-        const toolbox        = workspace.toolbox_; // eslint-disable-line
-        this.is_toolbox_open = !this.is_toolbox_open;
+        const workspace        = Blockly.derivWorkspace;
+        const toolbox          = workspace.toolbox_; // eslint-disable-line
```

Comment:
We can use the public `getToolbox` function rather than accessing the private property here (I've changed this in one of my PRs).